Tasks/Korotkevich_Grigory/Task_3/Task3_1st part/Task3_1st part .py

```diff
@@ -0,0 +1,16 @@
+from collections import Counter
```

Comment:
why did you hesitate to use it?)

Reply:
I tried different options, this one somehow didn’t finish. And I decided to go in search)) I tried further and came across (lambda).